### PR TITLE
chore: Bump Snaps packages

### LINKED ISSUE #2776: Expose a component that displays an Avatar (Jazzicon/Blockie)
Custom UI could use a Component that displays a Jazzicon/Blockie of an address.

Create a component that takes a CAIP-10 address and displays the Jazzicon or Blockie (depending on the extension's preference).

### PR DESCRIPTION
**Pull request by Mrtenz**
## **Description**

This bumps all Snaps packages to the latest versions.

Summary of changes in the snaps deps:
- Allow updating `context` in `snap_updateInterface`
- Add `snap_getCurrencyRate` RPC method
- Add `Avatar` component
- Add `min`, `max` and `step` props to `Input` component
- Add `size` prop to `Heading` component
- Add support for `metamask:` schemed URLs in Snap UI links
- Pass full URLs to PhishingController
- Ignore Snap insight response if transaction or signature has already been signed
- Allow `Link` in `Row` and `Address` in `Link`

Closes https://github.com/MetaMask/snaps/issues/2776

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/27376?quickstart=1)

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by Mrtenz** [comment]
@metamaskbot update-policies

File: builds.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ buildTypes:
       - SEGMENT_WRITE_KEY_REF: SEGMENT_PROD_WRITE_KEY
       - ALLOW_LOCAL_SNAPS: false
       - REQUIRE_SNAPS_ALLOWLIST: true
-      - IFRAME_EXECUTION_ENVIRONMENT_URL: https://execution.metamask.io/iframe/6.7.0/index.html
+      - IFRAME_EXECUTION_ENVIRONMENT_URL: https://execution.metamask.io/iframe/6.9.0/index.html
```
Comment (FrederikBolding) [inline builds.yml:29]: Missing a bump on this one

**Review by FrederikBolding — COMMENTED** [review]
Can we add a changelog to the PR that lists the added features etc in this bump?

Makes it easier to track down when features got released.

**Review by GuillaumeRx — DISMISSED** [review]

**Comment by Mrtenz** [comment]
@metamaskbot update-policies

**Review by GuillaumeRx — DISMISSED** [review]

**Review by ritave — DISMISSED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Review by GuillaumeRx — APPROVED** [review]

**Comment by bowensanders** [comment]
LGTM ✅

**Pull request merged**